pkgs/development/python-modules/djangorestframework/default.nix

```diff
@@ -1,29 +1,30 @@
-{
-  lib,
-  buildPythonPackage,
-  fetchFromGitHub,
-  coreapi,
-  django,
-  django-guardian,
-  pythonOlder,
-  pytest-django,
-  pytest7CheckHook,
-  pytz,
-  pyyaml,
-  uritemplate,
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, coreapi
+, django
+, django-guardian
+, inflection
+, pythonOlder
+, pytest-django
+, pytest7CheckHook
+, pytz
+, pyyaml
+, uritemplate
+,
```

Comment:
This diff has nothing to do with the version update commit, please split it to another commit.

Reply:
We recently reformatted all packages with `nixfmt-rfc-style`, and we'd very much like to keep that style to prevent noise caused by reformatting.